projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -56,6 +57,7 @@ export class MediaViewerComponent implements OnChanges, AfterContentInit {
 
   ngAfterContentInit() {
     this.setToolbarButtons();
+    this.toolbarEventsOutput.emit(this.toolbarEvents);
```

Comment:
Having something emitted just once on init feels a bit wrong but I guess that's the way we decided to implement this.

Reply:
yh, i wasnt sure the best approach for this, i remember talking to you about it. i believe this is just so they can get the service so once is effort in my opinion.